project/Lianghy0326/Readme.md

```diff
@@ -0,0 +1,49 @@
+# DE
+Automatic Differential Equation Solver Optimizer
+
+
+## Basic Information
+Github Repository: https://github.com/Lianghy0326/DE
+
+
+Design an intelligent C++ solver that automatically selects the optimal solving method for differential equation (DE) problems. 
```

Comment:
This is way longer than 79 chars.  You should configure your editor to format your text.

You have most of the lines to be short enough for review, so it's not too bad.  Please fix the line width when you update for other things.